src/Compilers/Core/Portable/AdditionalTextFile.cs

```diff
@@ -42,7 +42,7 @@ public AdditionalTextFile(CommandLineSourceFile sourceFile, CommonCompiler compi
         /// Returns a <see cref="SourceText"/> with the contents of this file, or <c>null</c> if
         /// there were errors reading the file.
         /// </summary>
-        public override SourceText GetText(CancellationToken cancellationToken = default(CancellationToken))
+        public override SourceText GetText(CancellationToken cancellationToken = default)
```

Comment:
The 15.3 preview is publicly available now and does not break with this change.  The consensus among the leads is that it is reasonable to expect contributors to Roslyn to be on the public preview track.  Especially in the time surrounding releases. 

@Pilchie can correct me if my memory is wrong here.